Docs for Enterprise Hub (#1081)
huggingface/hub-docs: docs/hub/billing.md

All users and organizations accounts have a billing system set up. You can submit your payment info and then access "pay-as-you-go" services.

> From the [Settings > Billing](https://huggingface.co/settings/billing) page, you can see a real time view of your usage across different HF services:

Reviewer: ```suggestion
From the [Settings > Billing](https://huggingface.co/settings/billing) page, you can see a real time view of your paid usage across all HF services, for instance:
```

- [Spaces](./spaces),
- [AutoTrain](https://huggingface.co/docs/autotrain/index),
- [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index)

> - [PRO subscriptions](https://huggingface.co/pricing) (for users)

Reviewer: ```suggestion
- [PRO subscription](https://huggingface.co/pricing) (for users)
```

> - [Enterprise Hub subscriptions](https://huggingface.co/enterprise) (for orgs)

Reviewer: ```suggestion
- [Enterprise Hub subscription](https://huggingface.co/enterprise) (for organizations)
```
